plugins/dataframe-gradle-plugin/src/main/kotlin/org/jetbrains/dataframe/gradle/ConvenienceSchemaGeneratorPlugin.kt

```diff
@@ -45,29 +57,25 @@ class ConvenienceSchemaGeneratorPlugin : Plugin<Project> {
                             false
                         }
                     }
-                val mainKspCfg = if (isMultiplatform) "kspJvm" else "ksp"
-                val testKspCfg = if (isMultiplatform) "kspJvmTest" else "kspTest"
-                try {
-                    target.configurations.getByName(mainKspCfg).dependencies.add(
-                        target.dependencies.create(
-                            "org.jetbrains.kotlinx.dataframe:symbol-processor-all:$preprocessorVersion",
-                        ),
-                    )
-                } catch (e: UnknownConfigurationException) {
-                    target.logger.warn(
-                        "Configuration '$mainKspCfg' not found. Please make sure the KSP plugin is applied.",
-                    )
                 }
-                try {
-                    target.configurations.getByName(testKspCfg).dependencies.add(
-                        target.dependencies.create(
-                            "org.jetbrains.kotlinx.dataframe:symbol-processor-all:$preprocessorVersion",
-                        ),
-                    )
-                } catch (e: UnknownConfigurationException) {
-                    target.logger.warn(
-                        "Configuration '$testKspCfg' not found. Please make sure the KSP plugin is applied.",
-                    )
+                val overriddenConfigs = target.findProperty(PROP_KSP_CONFIGS)?.let { (it as String) }?.split(",")
```

Comment:
Maybe we should also trim the entries in the list. When a user writes: "config1, config2", we now get the configs "config1" and " config2". I'm not sure `getByName` can find it like that

Reply:
I'll add it quickly so we can merge :)